integrationtests/src/test/java/io/kroxylicious/proxy/KafkaClusterIT.java

```diff
@@ -152,4 +230,10 @@ private void createTopic(AdminClient admin1, String topic, short replicationFact
     void before(TestInfo testInfo) {
         this.testInfo = testInfo;
     }
+
+    @AfterEach
+    void after() throws IOException {
+        Path filePath = Paths.get(KeytoolCertificateGenerator.getCertLocation());
+        Files.deleteIfExists(filePath);
```

Comment:
Should we do this after every test method or just after the class?

Reply:
The jks file is created for both server and client so we should wait until the client uses it to remove it. So instead of doing the removal after the client uses it, I put it here in case we want to use it more than once in the same test.